pkg/runtime/wfengine/backend.go

```diff
@@ -172,10 +182,14 @@ func (be *actorBackend) AddNewOrchestrationEvent(ctx context.Context, id api.Ins
 		return err
 	}
 
+	var actorType string
+	if be.config != nil {
+		actorType = be.config.WorkflowActorType
```

Comment:
is empty string a valid actor type? 
Should an error be returned if config is nil?

Reply:
Was Doing that only, haven't yet requested re-review

Reply:
I am done with the changes, hope this should be fine now